src/main/scala/main.scala

```diff
@@ -2,6 +2,10 @@ package kaptenallocweb
 
 import org.scalajs.dom
 import org.scalajs.dom.document
+import kaptenallocweb.ics.*
+import org.scalajs.dom.Blob
+import org.scalajs.dom.BlobPropertyBag
```

Comment:
If these are needed we could use the direct path instead. 

org.scalajs.dom is already imported.

Reply:
Should be resolved, see https://github.com/bjornregnell/kapten-alloc-web/pull/4#issuecomment-975295391